src/thorchain-exporter/Exporter.ts

```diff
@@ -123,7 +139,9 @@ export class Exporter {
             expectedExportCount += rangeTxs.length;
 
             // Output all txs in each range to CSV
-            writeCsv(`${outputPath}/all-${range.from}_${range.to}.csv`, rangeTxs);
+            const fn1 = 'all-' + range.from + '_' + range.to + '.csv';
```

Comment:
I prefer template strings when there is more than one variable

```suggestion
            const fn1 = `all-${range.from}_${range.to}.csv`;
```